docs/Makefile

```diff
@@ -12,7 +12,11 @@ BUILDDIR      = _build
 help:
 	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
 
-.PHONY: help Makefile
+environment:
+	conda create --name conda-docs --channel defaults python=3.8 --yes
+	conda run --name conda-docs pip install -r requirements.txt
```

Comment:
```suggestion
	conda run --name conda-docs pip install -r docs/requirements.txt
```

Reply:
hm. this is the Makefile in the docs folder ... prefixing with docs/ wont work here.

Reply:
Wait, this isn't right, I missed that this is in the docs directory. Could we instead use the parent directory Makefile?

Reply:
we could also move the target to the Makefile in the repo root and call the target env-docs instead.

Reply:
ok. wil do.